run-in-retro-term.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# cool-retro-term support for tetris.sh
+# Author: Rojen Zaman <rojen@riseup.net>
+# License: WTFPL
+
+set -u                                                    # return error if string null
+stdout="/dev/null"                                        # default stdout, optional verbose mode is /dev/stdout
+script_dir=$(cd $(dirname $0) && pwd)                     # script dir, it is require for cool-retro-term
+
+# usage
+usage() {
```

Comment:
Do you think we can use here printf approach to make usage test properly indentated?

Reply:
why do you not use cat<<EOT or cat <<EOF

it is so readable

Reply:
I fully agree with you and I really like heredocs. The problem is that you can nicely fit them into functions:
```
function_with_heredoc() {
    bla
    foo
    bar
    cat <<EOF
something
we 
want
to
print
EOF
    check
    cleanup
    return
}
```
This is not nicely formatted and if your heredoc is long it negatively affect perception of your script. If your script has no functions I see absolutely no reasons not to use heredoc. Of course even with functions you can do some post processing like:
```
func() {
    cat <<"    EOF"|sed -e 's/^        //'
        11 22
        33 44
    EOF
}
```
but this is too hacky even for my taste :).

Reply:
thanks a lot for your explanation

i wanted to use this :)
`"    EOF"|sed -e 's/^        //'`

Reply:
lol vscode couldn't detect it

Reply:
and in that, "$0" not work

Reply:
Ohhh, yes, this is because of the quoted delimiter. I tried
```
cat <<\ \ \ \ EOF|sed -e 's/^        //'
```
this works, but $ variables are not expanded. Well, we can use this:
```
func2() {
    cat <<EOF|sed -e 's/^        //'
        this is $0
        11 22
        33 44
EOF
```
as a compromise :)